lib/internal/process/promises.js

```diff
@@ -41,88 +41,179 @@ const { isErrorStackTraceLimitWritable } = require('internal/errors');
 // *Must* match Environment::TickInfo::Fields in src/env.h.
 const kHasRejectionToWarn = 1;
 
-const maybeUnhandledPromises = new SafeWeakMap();
-const pendingUnhandledRejections = [];
-const asyncHandledRejections = [];
-let lastPromiseId = 0;
+// By default true because in cases where process is not a global
```

Comment:
Why would process not be a global?

Reply:
IIRC when you instantiate it as a Worker.

Reply:
process is a global in workers

Reply:
There was a Case were process was not a global. Some tests broke. Maybe it was even at startup time.

I will investigate it again and give you proper feedback.

Reply:
see below :)